_includes/markdown/JavaScript.md

```diff
@@ -143,4 +143,6 @@ There are many JavaScript libraries available today. Many of them directly compe
 
 [React](https://reactjs.org/) - Using React provides a library to create large-scale, stateful JavaScript applications. It aims to provide a flexible system of creating highly componentized user interfaces. [Learn more about how we use React]({{ site.baseurl }}/react).
 
+[Vue](https://vuejs.org/) - Implementing Vue on a project allows us to take advantage of the statefulness built into something like React, but apply it on a much more lightweight and smaller scale as to not bog down performance by loading in a heavy library. [Learn more about how we use Vue]({{ site.baseurl }}/vue).
```

Comment:
How "heavy" is React? Is this really a major concern today?

Reply:
It's pretty heavy, you also need React DOM and either a full webpack set up or a JSX converter. So yeah, it's a pretty significant thing to add into a project when held up against Vue where you don't need anything other than the library itself.